app/services/hyrax/collections/collection_member_query_service.rb

```diff
@@ -0,0 +1,91 @@
+module Hyrax
+  module Collections
+    # Responsible for retrieving collection members
+    class CollectionMemberQueryService
```

Comment:
Can we rename to `CollectionMemberService`? This is returning actual members, not a query used to retrieve members, so I think that's more clear.

Reply:
done.